semver/greater_than_range_test.ts

```diff
@@ -167,3 +168,41 @@ Deno.test("greaterThanRange() checks if the semver is greater than the range", a
     });
   }
 });
+
+Deno.test("greaterThanRange() handles equals operator", () => {
+  const version = {
+    major: 1,
+    minor: 0,
+    patch: 0,
+    prerelease: [],
+    build: [],
+  };
+  const range = [[{
+    operator: "=" as unknown as Operator,
+    major: 1,
+    minor: 0,
+    patch: 0,
+    prerelease: [],
+    build: [],
+  }]];
+  assertEquals(greaterThanRange(version, range), false);
+});
+
+Deno.test("greaterThanRange() handles not equals operator", () => {
+  const version = {
+    major: 1,
+    minor: 0,
+    patch: 0,
+    prerelease: [],
+    build: [],
+  };
+  const range = [[{
+    operator: "!=" as unknown as Operator,
+    major: 1,
+    minor: 0,
+    patch: 0,
+    prerelease: [],
+    build: [],
+  }]];
+  assertEquals(greaterThanRange(version, range), true);
```

Comment:
I wonder if this result is correct

I also just realized `!=` is not a valid operator in `npm:semver` (`semver.gtr("1.0.0", "!=1.0.0")` throws with invalid operator error, and docs doesn't mention `!=` https://github.com/npm/node-semver?tab=readme-ov-file#ranges

I'd suggest we should skip this particular test case for now.